spec/lib/whitehall_importer/change_history_spec.rb

```diff
@@ -0,0 +1,81 @@
+RSpec.describe WhitehallImporter::ChangeHistory do
+  let(:first_edition) do
+    build(:whitehall_export_edition,
+          state: "superseded",
+          revision_history: [build(:whitehall_export_revision_history_event,
+                                   state: "published",
+                                   created_at: Time.zone.now.last_year.rfc3339)])
```

Comment:
Do we use this at all?

Reply:
Nope we don't, added it originally so the first edition would have a published at time which was before the third edition lets say, have removed it anyway as it's superfluous.